float32 x
float32 y
float32 z
float32 width
float32 height
sensor_msgs/PointCloud2 whole_pointcloud
---
sensor_msgs/PointCloud2 object_pointcloud
